packages/just-scripts/src/tasks/apiExtractorTask.ts

```diff
@@ -1,14 +1,15 @@
 import { logger, TaskFunction } from 'just-task';
-import fs from 'fs';
-import glob from 'glob';
+import fs from 'fs-extra';
 import path from 'path';
 import { tryRequire } from '../tryRequire';
+// This is a TYPING ONLY import from <root>/typings/api-extractor.d.ts
+import * as ApiExtractorTypes from '@microsoft/api-extractor';
```

Comment:
do you know what happens if you don't have this installed as a typescript consumer of this?

Reply:
It's still fine. The typing import resolves to that new .d.ts file I added.

Reply:
Oh and I made sure not to use any of the types in anything exported, so it also won't cause problems if you're a typescript consumer who has a different version installed.